load-test-framework/src/main/proto/command.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+package pubsub;
+
+option java_package = "com.google.pubsub.flic.common";
+option java_outer_classname = "Command";
+
+service LoadtestFramework {
+  rpc StartClient(CommandRequest) returns (google.protobuf.Empty) {
+  }
+}
+
+message CommandRequest {
+  google.protobuf.Timestamp start_time = 1;
```

Comment:
Please add comments/documentation to these fields. In particular, please address these points:
- What happens if start_time is in the past? What about stop_time?
- What happens if start_time >= stop_time?